tink/api/v1alpha1/hardware_types.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// HardwareIDAnnotation is used by the controller to store the
+// ID assigned to the workflow by Tinkerbell.
```

Comment:
```suggestion
// HardwareIDAnnotation is used by the controller to store the
// hardware ID assigned to the workflow by Tinkerbell.
```

Reply:
Ah, copypasta issue. Should have been 'ID assigned to the hardware by Tinkerbell'